intro-section.tex

```diff
@@ -75,9 +75,12 @@ \subsection{Hello World in cylc}
 \terminology{A cylc \underline{suite daemon} is a light-weight server program
 dedicated to managing a single workflow.}
 
-\paragraph*{Demo}
-Copy the Hello World example code into a suite.rc file in a directory called
-hello\_world, and run the following commands:
+\begin{shaded*}
+
+\subsection{Hello World Tutorial}
```

Comment:
The shaded environment doesn't seem to play nicely with headings (it shades the vertical space before the heading).

Visually this works ok:

```latex
\vspace{5mm}
\begin{shaded*}
{\textbf{\large{Hello World Tutotial}}}
```
... but obviously doesn't appear in the document structure.